stumpy/gpu_ostinato.py

```diff
@@ -94,19 +94,19 @@ def gpu_ostinato(Ts, m, device_id=0, normalize=True, p=2.0):
     if not isinstance(Ts, list):  # pragma: no cover
         raise ValueError(f"`Ts` is of type `{type(Ts)}` but a `list` is expected")
 
+    Ts_subseq_isconstant = [None] * len(Ts)
     M_Ts = [None] * len(Ts)
     Σ_Ts = [None] * len(Ts)
     for i, T in enumerate(Ts):
+        Ts_subseq_isconstant[i] = core.rolling_isconstant(T, m)
         Ts[i], M_Ts[i], Σ_Ts[i] = core.preprocess(T, m)
```

Comment:
> Are there any cases where we would call `core.preprocess` without needing to call `rolling_isconstant`? If it is rare (i.e., more than 50% of the time we will always need `T_subseq_isconstant`) then we should simply add `rolling_isconstant` into `core.preprocess` and return `T_subseq_isconstant`

Great suggestion!  I will check out the current changes just to make sure I am not missing anything. I guess I will eventually add it to the `core.preprocess(...)`. Another good thing about this is that I do not need to be worried about whether `T` is already preprocessed or not. (because, we need the original `T` to get the `T_subseq_isconstant`). So, if we put it inside the `core.preprocess`, we know that things are okay because the input `T` being passed to `core.preprocess` is the original one.